README.md

```diff
@@ -32,8 +32,10 @@ php app/console do:mi:version --add 20150515104913
 
 running development machine
 ---------------------------
-* vagrant up
-* vagrant ssh
-* cd /srv/www
-* bin/console fo:us:cr (follow the steps)
+Prerequirements: Vagrant installed, Virtualbox installed, Ansible installed
+
+* ```vagrant up```
```

Comment:
does this even render in Markdown? some combination of inline code block and fenced code block? to see what I was talking about? https://raw.githubusercontent.com/tolry/talkly/master/README.md